test/metabase/test/data/duckdb.clj

```diff
@@ -1,36 +1,38 @@
 (ns metabase.test.data.duckdb
   (:require
-   [clojure.edn :as edn]
    [clojure.java.io :as io]
-   [clojure.string :as str]
    [metabase.config :as config]
    [metabase.driver :as driver]
+   [metabase.driver.sql-jdbc.connection :as sql-jdbc.conn]
+   [metabase.driver.sql-jdbc.execute :as sql-jdbc.execute]
+   [metabase.driver.sql-jdbc.sync.describe-table-test :as describe-table-test]
    [metabase.test.data.interface :as tx]
    [metabase.test.data.sql :as sql.tx]
    [metabase.test.data.sql-jdbc :as sql-jdbc.tx]
-   [metabase.test.data.sql-jdbc.execute :as execute]
    [metabase.test.data.sql-jdbc.load-data :as load-data]
-   [metabase.test.data.sql.ddl :as ddl]
-   [metabase.util :as u]
-   [metabase.util.log :as log]))
+
+   [metabase.test.data.sql.ddl :as ddl]))
 
 (set! *warn-on-reflection* true)
 
 (sql-jdbc.tx/add-test-extensions! :duckdb)
 
-(doseq [[feature supported?] {:foreign-keys  (not config/is-test?) 
-                              :upload-with-auto-pk (not config/is-test?)}]
+(doseq [[feature supported?] {:foreign-keys  (not config/is-test?)
```

Comment:
should the limitations/exceptions made for the test be documented somewhere? CONTRIBUTING.md or something?